docs/node/run-your-node/advanced/sync-node-using-state-sync.md

```diff
@@ -128,24 +106,3 @@ Query our public Rosetta Gateway instance and obtain the trusted height and hash
 Query our public Oasis Node's gRPC endpoint and obtain the trusted height and hash there:
 
 1. _TODO._
-
```

Comment:
I know this probably isn't your markdown, but what's with the TODOs above? The public gRPCs we use for our wallets won't work?

Reply:
Probably a TODO for the example (e.g. show how to query the gRPC endpoint to obtain the height and hash). I can add it.

Reply:
Updated